HW_3/task16.py

```diff
@@ -0,0 +1,15 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих строках записаны N целых чисел Ai. Последняя строка содержит число X
+# Пример:
+# N = 5
+# 1 2 3 4 5
+# 3 -> 1
+n = int(input("Введите величину массива: "))
+a = list(range(1, int(n) + 1))
+x = int(input("Введите число: "))
+count = 0
+for i in range(len(a)):
```

Comment:
У вас уже есть параметры, содержащие в длину списка
```suggestion
for i in range(n):
```